channels-rke2.yaml

```diff
@@ -890,6 +890,26 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1500
     featureVersions: *featureVersions-v1
+  - version: v1.23.17+rke2r1
+    minChannelServerVersion: v2.6.4-alpha1
+    maxChannelServerVersion: v2.7.99
```

Comment:
We've been doing `maxChannelServerVersion: v2.6.99` for 2.6 releases except of `v1.23.16+rke2r1` and `v1.24.10+rke2r1` where it was accidentally set to `v2.7.99`. I think we want to go back to setting it to `v2.6.99` for consistency.